package.json

```diff
@@ -7,7 +7,8 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "format": "prettier --check ."
+    "format": "prettier --check .",
+    "prepare": "husky install"
```

Comment:
Husky is just a tool to install [Git Hooks](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks). By adding the [`prepare`](https://docs.npmjs.com/cli/v8/using-npm/scripts#prepare-and-prepublish) script above, we ensure that anyone that installs our project will install the hooks configured under the `.husky` directory.